projects/ad7616_sdz/zed/system_bd.tcl

```diff
@@ -9,5 +9,13 @@ ad_ip_parameter rom_sys_0 CONFIG.ROM_ADDR_BITS 9
 
 sysid_gen_sys_init_file
 
+# system level parameters
+set SI_OR_PI $ad_project_params(SI_OR_PI)
+
+adi_project_files ad7616_sdz_zed [list \
+	"../../../library/common/ad_edge_detect.v" \
+	"../../../library/util_cdc/sync_bits.v" \
```

Comment:
End the project files list after the last element in this way:
`"../../../library/util_cdc/sync_bits.v"]`

Reply:
Done.